scripts/zones/The_Ashu_Talif/mobs/Ashu_Talif_Captain.lua

```diff
@@ -32,10 +55,10 @@ entity.onMobFight = function(mob, target)
 end
 
 entity.onMobRoam = function(mob)
-    local jumped = mob:getLocalVar('jump')
+    -- failsafe in case of party wipe/reraise
+	local jumped = mob:getLocalVar('jump')
```

Comment:
I think it would be smart to make a local function and call it in `onMobRoam` and `onMobEngaged`

In the top of the script define:
```
local captainEngageSequence(mob)
    local jumped = mob:getLocalVar('jump')
    local ready = mob:getLocalVar('ready')

	-- Becomes ready when the Crew is engaged. Jump down!
    if ready == 1 and jumped == 0 then
        mob:setLocalVar('jump', 1)
        mob:showText(mob, ID.text.OVERPOWERED_CREW)
        mob:hideName(true)
        mob:entityAnimationPacket('jmp0')
        mob:timer(2000, function(m)
            m:setPos(0, -22, 13, 192)
            m:entityAnimationPacket('jmp1')
            m:showText(mob, ID.text.TEST_YOUR_BLADES)
            m:timer(2000, function(mAnimation)
                mAnimation:hideName(false)
                mAnimation:setUntargetable(false)
            end)
        end)
    end
end
```

Then in this section and `onMobEngaged`, remove everything in the function and just have:
```suggestion
    -- failsafe in case of party wipe/reraise
    captainEngageSequence(mob)
```